scipy/optimize/tests/test_quadratic_assignment.py

```diff
@@ -197,6 +197,16 @@ def test_options(self):
                                    options={'P0': K})
         assert_(11156 <= res.fun < 21000)
 
+        # test padded qap
+        n = 50
+        p = 0.4
+        G1 = _er_matrix(n, p)
+        G2 = G1[: (n - 1), : (n - 1)]  # remove two nodes
```

Comment:
```suggestion
        G2 = G1[: -1, -1]  # remove two nodes
```
Doesn't this remove one node?